### ISSUE TITLE
Create index.html and 404.html for storage buckets

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/CRBM-Viz#4043]
- [x] https://files.biosimulations.dev/index.html
- [x] https://files.biosimulations.dev/404.html
- [x] https://files.biosimulations.org/index.html
- [x] https://files.biosimulations.org/404.html
- [x] https://static.biosimulations.org/404.html

**Comment by jonrkarr** [comment]
Done
- Committed files to https://github.com/biosimulations/static/tree/dev/files.
- Uploaded files to the UConn buckets with keys `index.html` and `404.html`.

**Comment by jonrkarr** [comment]
The files are externally accessible at URLs such as https://files.biosimulations.org/s3/index.html and https://files.biosimulations.org/s3/404.html.

However, `index.html` and `404.html` don't render when the following URLs are fetched.
- https://files.biosimulations.org/
- https://files.biosimulations.org/s3/
- https://files.biosimulations.org/undefined
- https://files.biosimulations.org/s3/undefined
- https://files.biosimulations.dev/
- https://files.biosimulations.dev/s3/
- https://files.biosimulations.dev/undefined
- https://files.biosimulations.dev/s3/undefined

Todo
- [x] Copy files to new Google buckets
- [ ] Configure the files URL mapping to render `index.html` and `404.html` (i.e., remove the `s3/` from the files URLs). This can be done as part of moving to a Google bucket.

**Comment by bilalshaikh42** [comment]
These are to be placed on the google bucket, not the UConn bucket. files.biosimulations.dev currently points to the new google bucket.

**Comment by bilalshaikh42** [comment]
I uploaded the pages for the files.biosimulations.dev domains so those should now work. I also added the files for the prod bucket, though the URL files.biosimulations.org is still pointing at Uconn and not google cloud, so this won't work yet.


> * Configure the files URL mapping to render `index.html` and `404.html` (i.e., remove the `s3/` from the files URLs). This can be done as part of moving to a Google bucket.

I am not sure what you mean by this. The ./s3 prefix is just due to the way the UConn loadbalancer works. The prefix does not apply to the google cloud buckets. I am not sure where this would be removed from (configuration?)

**Comment by jonrkarr** [comment]
I realize the HTML files are intended for the new buckets. I wasn't sure how far you've gotten setting them up so I copied them to the old buckets to start.

Looks like you copied them to the new dev bucket. The index and 404 files are working now.

I figured the `s3/` prefix is related to UConn's deployment. I believe this only appeared in the configuration for the external bucket URL. I assume this will go away with moving from UConn to Google buckets.
